subsys/bluetooth/audio/has.c

```diff
@@ -1200,6 +1289,84 @@ int bt_has_register(const struct bt_has_register_param *param)
 		has.features |= BT_HAS_FEAT_WRITABLE_PRESETS_SUPP;
 	}
 
+	return 0;
+}
+
+int bt_has_features_set(const struct bt_has_features_param *features)
+{
+	uint8_t tmp_features;
+	int err;
+
+	/* Check whether any features will change, otherwise we don't want to notify clients */
+	if (!((has.features & BT_HAS_FEAT_HEARING_AID_TYPE_LSB) ^
```

Comment:
Should it be possible to call this before calling `bt_has_register`?

Reply:
Good point, it should not